backend/data_tools/src/disable_users/queries.py

```diff
@@ -0,0 +1,25 @@
+EXCLUDED_USER_IDS = [520, 521, 522, 523, 525, 526]
```

Comment:
These user ids will only reference fake users in lower environments - in PROD these will be real users.  It would be better to use the oidc ids here to reference them.  Same for the `SYSTEM_USER_ID` - that will only be the correct id in lower environments where test data is being loaded.